src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -125,6 +139,22 @@ private String extractStudentRegistrationKey(StudentAttributes student) {
         return joinLink.substring(startIndex);
     }
 
+    private String isolatePrevPageUrl() {
+        String stringTillLastSlash = currentUrl.substring(0, currentUrl.lastIndexOf('/'));
+        int indexOfSecondLastSlash = stringTillLastSlash.lastIndexOf('/');
+
+        return currentUrl.substring(indexOfSecondLastSlash, currentUrl.indexOf('?'));
+    }
+
+    private boolean compareRefererAndCurrentPage(String current) {
```

Comment:
If I'm not mistaken, the argument here shouldn't be called `current`, because it **does not** represent the current url of the page right?